core/src/main/java/com/sequenceiq/cloudbreak/reactor/DecommissionHandler.java

```diff
@@ -132,11 +132,25 @@ public void accept(Event<DecommissionRequest> event) {
                     .map(hostsToRemove::get)
                     .filter(Objects::nonNull)
                     .collect(Collectors.toList());
-            decommisionedInstances.forEach(clusterDecomissionService::deleteHostFromCluster);
-            updateInstanceStatuses(decommisionedInstances, InstanceStatus.DECOMMISSIONED, "instance successfully downscaled");
-            clusterDecomissionService.restartStaleServices();
+            String decommisionResult = decommisionedInstances.parallelStream().map(metaData -> {
+                try {
+                    clusterDecomissionService.deleteHostFromCluster(metaData);
+                } catch (Exception e) {
+                    return e.getMessage();
```

Comment:
this should contain the instance id and/or fqdn also, and not only the message, so we could identify easily which node failed